### ISSUE TITLE
tslint throws "jsdoc is not formatted correctly" only on Windows for typedoc blank asterisk

### ISSUE THREAD

**Issue by pellekrogholt** [palantir/tslint#324]
## versions in use

- ts 1.4.1
- grunt-ts: 3.0.0
- grunt-tslint: 2.0.0
- tslint 2.1.1
- typedoc 0.1.0

## problem 

i have typedoc here and there - some times using a blank line for readability or because markdown requires it - lets have an example:

    /*
     * Test file
     *
     * Test a file documentation text
     *
     *     Some code that should be markdown code formatted
     *
     * @param test    Some text about test param.
     * @param path    Some text about path param.
     */
    function testFile(test: any, path:any) {
    ...

on OS like ubuntu 14.* or osx mavericks this passes with my tslint task

    grunt tslint:all

but on Windows (Windows 8.0 / Windows Server 2012) the empty `*` lines raises

{code}
jsdoc is not formatted correctly 
{code}

i could change the empty lines with `* ` and the i pass. but i'm not super happy about this because various white space cleaners cleans that additional extra space away.

pardon me if I post the issue in the wrong spot. i'm a bit unsure if this is a tslint or grunt-tslint issue or me that are one a wrong path here. 

thx!

//

pelle

**Comment by mwe** [comment]
I raised probably the same issue, If your problem is related to UTF8 files. ( see #324 )

**Comment by eonarheim** [comment]
I believe the problem is this line of code right here when we split the lines on "\n" windows line endings are "\r\n"

```js
private findFailuresForJsdocComment(commentText: string, startingPosition: number, sourceFile: ts.SourceFile) {
        var currentPosition = startingPosition;
        var lines = commentText.split("\n"); // Windows file line endings are \r\n!
        var jsdocPosition = currentPosition;
        var firstLine = lines[0];
...
```
which then causes this regex to fail, since there is an extra "\r" from the split

```js
// regex is: start of string, followed by any amount of whitespace, followed by *,
// followed by either a space or the end of the string
var asteriskMatch = line.match(/^\s*\*( |$)/);
if (asteriskMatch == null) {
    this.addFailureAt(jsdocPosition, line.length, Rule.FORMAT_FAILURE_STRING);
}
```

I have raised a pull request fixing this with tests :)

**Comment by adidahiya** [comment]
looks like the fix got merged. please comment if this bug still occurs in the latest tslint.
